### PR TITLE
feat(crons): Add component for rendering processing error title

### PR DESCRIPTION
**Pull request by davidenwang**
Renders a title that would eventually go into a component looking like this:

image

### CODE REVIEW

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import {t} from 'sentry/locale';
+import {ProcessingErrorType} from 'sentry/views/monitors/types';
+
+export function ProcessingErrorTitle({type}: {type: ProcessingErrorType}) {
+  switch (type) {
+    case ProcessingErrorType.CHECKIN_ENVIRONMENT_MISMATCH:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:7]:
```suggestion
      return t('Environment mismatch');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.CHECKIN_FINISHED:
+      return t('Check-In has already succeeded or failed.');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:9]:
```suggestion
      return t('Check-in already completed');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.CHECKIN_GUID_PROJECT_MISMATCH:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:11]:
```suggestion
      return t('DSN project mismatch');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.CHECKIN_INVALID_DURATION:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:13]:
```suggestion
      return t('Invalid duration');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.CHECKIN_INVALID_GUID:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:15]:
```suggestion
      return t('Invalid GUID');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.CHECKIN_VALIDATION_FAILED:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:17]:
```suggestion
      return t('Invalid check-in payload');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.MONITOR_DISABLED:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:19]:
```suggestion
      return t('Monitor disabled');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.MONITOR_DISABLED_NO_QUOTA:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:21]:
```suggestion
      return t('Insufficient quota');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.MONITOR_INVALID_CONFIG:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:23]:
```suggestion
      return t('Invalid monitor payload');
```

File: static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    case ProcessingErrorType.MONITOR_INVALID_ENVIRONMENT:
+      return t('<placeholder description>');
```
Comment (gaprl) [inline static/app/views/monitors/components/processingErrors/processingErrorTitle.tsx:25]:
```suggestion
      return t('Invalid environment');
```

**Review by gaprl — DISMISSED** [review]
I'd figured since we already have a detailed explanation in each processing error, we'd keep these titles short and concise for easy reading.

**Review by evanpurkhiser — APPROVED** [review]

**Pull request merged**